### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by marvinhagemeister** [GoogleChrome/simplehttp2server#27]
OS: MacOS Sierra

When installing I always get an exception (found at https://github.com/developit/preact-cli/issues/26):

```bash
$ npm i simplehttp2server
> simplehttp2server@1.0.0 postinstall /Users/marvinhagemeister/dev/http2server/node_modules/simplehttp2server
> node lib/install.js

  ⚠ The `/Users/marvinhagemeister/dev/http2server/node_modules/simplehttp2server/vendor/simplehttp2server` binary doesn't seem to work correctly
  ⚠ simplehttp2server pre-build test failed
  ℹ compiling from source
  ✖ Error: mkdir -p /Users/marvinhagemeister/dev/http2server/node_modules/simplehttp2server/vendor && sh crosscompile.sh && mv ./0f3bf125-3c46-428e-94d8-d7546e50273f_darwin_amd64 /Users/marvinhagemeister/dev/http2server/node_modules/simplehttp2server/vendor/simplehttp2server
Command failed: sh crosscompile.sh && mv ./0f3bf125-3c46-428e-94d8-d7546e50273f_darwin_amd64 /Users/marvinhagemeister/dev/http2server/node_modules/simplehttp2server/vendor/simplehttp2server
simplehttp2server.go:24:2: cannot find package "github.com/NYTimes/gziphandler" in any of:
	/usr/local/Cellar/go/1.8.1/libexec/src/github.com/NYTimes/gziphandler (from $GOROOT)
	/Users/marvinhagemeister/go/src/github.com/NYTimes/gziphandler (from $GOPATH)
simplehttp2server.go:24:2: cannot find package "github.com/NYTimes/gziphandler" in any of:
	/usr/local/Cellar/go/1.8.1/libexec/src/github.com/NYTimes/gziphandler (from $GOROOT)
	/Users/marvinhagemeister/go/src/github.com/NYTimes/gziphandler (from $GOPATH)
simplehttp2server.go:24:2: cannot find package "github.com/NYTimes/gziphandler" in any of:
	/usr/local/Cellar/go/1.8.1/libexec/src/github.com/NYTimes/gziphandler (from $GOROOT)
	/Users/marvinhagemeister/go/src/github.com/NYTimes/gziphandler (from $GOPATH)
simplehttp2server.go:24:2: cannot find package "github.com/NYTimes/gziphandler" in any of:
	/usr/local/Cellar/go/1.8.1/libexec/src/github.com/NYTimes/gziphandler (from $GOROOT)
	/Users/marvinhagemeister/go/src/github.com/NYTimes/gziphandler (from $GOPATH)
simplehttp2server.go:24:2: cannot find package "github.com/NYTimes/gziphandler" in any of:
	/usr/local/Cellar/go/1.8.1/libexec/src/github.com/NYTimes/gziphandler (from $GOROOT)
	/Users/marvinhagemeister/go/src/github.com/NYTimes/gziphandler (from $GOPATH)

    at ChildProcess.exithandler (child_process.js:205:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:194:7)
    at maybeClose (internal/child_process.js:899:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
http2server@1.0.0 /Users/marvinhagemeister/dev/http2server
```

**Comment by surma** [comment]
Sadly, the [simplehttp2server npm package](https://www.npmjs.com/package/simplehttp2server) is not by me.

Please re-open this issue on the [respective repo](https://github.com/1000ch/simplehttp2server).

**Comment by kradharamanan-ias** [comment]
@marvinhagemeister I've posted a workaround here: https://github.com/developit/preact-cli/issues/73

**Comment by kamalasaurus** [comment]
@marvinhagemeister I've posted a workaround here:
https://github.com/developit/preact-cli/issues/73

**Comment by surma** [comment]
Actually, I have an idea on how to fix this on my end as well.

**Comment by surma** [comment]
I believe I have fixed this on my end. `npm i simplehttp2server` should work again :)

**Comment by marvinhagemeister** [comment]
Thanks for having a look. Just tried again today, but I get the same error as in the original post.

**Comment by surma** [comment]
Can you double-check if the problem is exactly the same, because it shouldn’t be. (Also, run `npm cache clean` if you haven’t already).

**Comment by marvinhagemeister** [comment]
@surma sure, just double checked with both `yarn`, `npm` and `node 7.10 + 8.0.0`. The issue is unfortunately reproducible for all combinations.

Maybe there is something wrong with the binaries itself? The first message is about the binaries not working correctly and than it proceeds to build fresh from source. I'm not familiar at all with go or how they generate the binaries.

```bash
The `/myproject/node_modules/simplehttp2server/vendor/simplehttp2server` binary doesn't seem to work correctly
```

**Comment by marvinhagemeister** [comment]
I'll close this one, since it's already tracked in https://github.com/1000ch/simplehttp2server/issues/2

**Comment by surma** [comment]
Yeah, I am aware. That message is a red herring. I can’t reproduce the failure at all.

 FWIW, I am working on an upstream fix at 1000ch/simplehttp2server#3.
